yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/Base64.java

```diff
@@ -75,7 +75,6 @@
  *
  *
  * @author  Xueming Shen
```

Comment:
Should we get rid of the Author tag at the same time?

Reply:
Discussed offline, keeping tag